### ISSUE TITLE
Bug: shallowMount fails on vue 3.2.32, works on 3.2.31

### ISSUE THREAD

**Issue by hjri** [vuejs/test-utils#1419]
**Describe the bug**

When using vue 3.2.31 everything works, but using 3.2.32 tests fail with errors like:
```
    X is undefined
    jo@webpack:///node_modules/@vue/test-utils/dist/vue-test-utils.esm-bundler.js:7984:5 <- index.js:37:13453
    Oo@webpack:///node_modules/@vue/test-utils/dist/vue-test-utils.esm-bundler.js:7993:12 <- index.js:37:13720
    @webpack:///test/unit/specs/components/rich_content.spec.js:52:21 <- index.js:49:498525
```

**To Reproduce**

use shallowMount with a jsx component?

https://git.pleroma.social/pleroma/pleroma-fe/-/jobs/208665

**Expected behavior**

works with vue 3.2.32

**Related information:**
- `@vue/test-utils` version: 2.0.0-rc.19
- `Vue` version: 3.2.32
- `node` version: v16.14.2
- `npm` (or `yarn`) version: 1.19.2

**Additional context**

**Comment by cexbrayat** [comment]
Hi @hjri 

This is indeed an issue, but we already have a PR to fix this #1418 
The fixed version should be released in the next hours.

I'll let the issue open until the release, in case someone else encounters it.

**Comment by cexbrayat** [comment]
VTU v2.0.0-rc.20 is now out with the support of Vue v3.2.32
